File: examples-salvo/cephyr/05_arrays.c
#include <stdio.h>
int
main (void)
{
  int a[4] = { 2, 4, 6, 8 };
  int total = 0;
  for (int i = 0; i < 4; ++i)
    total += a[i];
  printf ("%d\n", total);
  return 0;
}

File: examples-salvo/simple/delphia/05_arrays.pas
program Arrays;
var Values: array[1..4] of Integer;
    I, Total: Integer;
begin
  Values[1] := 2; Values[2] := 4; Values[3] := 6; Values[4] := 8;
  Total := 0;
  for I := 1 to 4 do Total := Total + Values[I];
  Writeln(Total);
end.
